src/formatters/deployResultFormatter.ts

```diff
@@ -117,6 +122,22 @@ export class DeployResultFormatter extends ResultFormatter {
     return get(this.result, 'response', {}) as MetadataApiDeployStatus;
   }
 
+  protected displayReplacements(): void {
+    if (this.isVerbose() && this.result.replacements?.size) {
```

Comment:
for deploy, I could go either way.  It *felt* noisy, like a thing I wouldn't want / pay attention to unless something weird was happening and I needed to investigate.

for push, they'll be there by default because it already has a `quiet` mode which omits them (from human and json).  Or at least that's what I meant.

Reply:
OK, deploy will show them by default.